caffe2/python/tutorials/MNIST.ipynb

```diff
@@ -553,6 +553,10 @@
     "workspace.RunNetOnce(train_model.param_init_net)\n",
     "# creating the network\n",
     "workspace.CreateNet(train_model.net, overwrite=True)\n",
+    "# Also initialize the test model before training\n",
+    "workspace.RunNetOnce(test_model.param_init_net)\n",
```

Comment:
Sure, here is the file generated by
`with open(os.path.join(root_folder, "test_init_net.pbtxt"), 'w') as fid:
    fid.write(str(test_model.param_init_net.Proto()))`
before `RunNetOnce` and `CreateNet`
[test_init_net.pb.txt](https://github.com/caffe2/caffe2/files/1079640/test_init_net.pb.txt)